package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@adyen/api-library",
-  "version": "2.1.8",
+  "version": "2.2.0",
   "description": "The Adyen API Library for NodeJS enables you to work with Adyen APIs and Hosted Payment Pages.",
```

```suggestion
  "description": "The Adyen API Library for NodeJS enables you to work with Adyen APIs.",
```
since you removed this in the README